Libbna/CUSTOM-CMS
# #279 : Category wise blogs on home page.

* Added a select dropdown menu on home screen which contains all the various categories of blogs.
* After selecting a category via select tag, the blogs related to the selected category is displayed on the screen.
* Testing steps : 
1. Go to home page.
2. Scroll down to `Category BLOGS` section.
3. Select any category from the drop-down.
4. The blogs specific to that category should get displayed without the screen getting reloaded.
